src/test/javascript/spec/component/exam/exam-scores/exam-scores.component.spec.ts

```diff
@@ -18,6 +18,8 @@ import { FaIconComponent } from '@fortawesome/angular-fontawesome';
 import { SortService } from 'app/shared/service/sort.service';
 import { JhiLanguageHelper } from 'app/core/language/language.helper';
 import { HelpIconComponent } from 'app/shared/components/help-icon.component';
+import { ParticipantScoresService, ScoresDTO } from 'app/shared/participant-scores/participant-scores.service';
+import * as _ from 'lodash';
```

Comment:
You are only using cloneDeep. Why are you importing the whole thing? 🤔

Reply:
Changed to named import in 8df83725325b424a16ff384e5130eda5e17e2565